app/Models/RequestPurchaseOrder.php

```diff
@@ -31,7 +32,39 @@ class RequestPurchaseOrder extends Model
         'transaction_date' => 'date',
         'metadata' => 'json',
         'approvals' => 'json',
+        'processing_status' => PurchaseOrderProcessingStatus::class,
     ];
+
+    public function getWorkflowAttribute()
+    {
+        return collect([
```

Comment:
remove collect()->toArray();
direct return []
implement the logic nga ako gi hatag 
REPAYMENT = [PENDING],
ISSUED = array_filter([!$this->is_prepayment ? PENDING : null, PREPAYMENT]),
ITEMS_RECEIVED = [ISSUED],
CHANGES = [ITEMS_RECEIVED],
TURNED_OVER = [ITEMS_RECEIVED, CHANGES],
POSTPAYMENT = [TURNED_OVER],
SERVED = array_filter([$this->is_prepayment ? TURNED_OVER : null, POSTPAYMENT])
]

I convert nalang ni kay pa bali man ni nga logic